pkg/kubelet/types/pod_update.go

```diff
@@ -23,6 +23,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	kubeapi "k8s.io/kubernetes/pkg/apis/core"
+	kubepod "k8s.io/kubernetes/pkg/kubelet/pod"
```

Comment:
Please run `./hack/update-bazel.sh` to add this new dependency to the proper `BUILD.bazel` file.

Reply:
> Thanks for this pr :)
> 
> I'm curious to see the results of the test suite after running the two `hack` scripts below. You may have an import cycle, in which case we may need to rethink the approach slightly. We'll know for sure after re-running the test suite after running the hack scripts.

I ran the above hack scripts. Doesn't look like it's helping with the import cycle though :(